ql/src/java/org/apache/hadoop/hive/ql/exec/FileSinkOperator.java

```diff
@@ -1204,8 +1205,10 @@ private void closeWriters(boolean abort) throws HiveException {
   private void closeRecordwriters(boolean abort) {
     for (RecordWriter writer : rowOutWriters) {
```

Comment:
So now there is a null check for the RecordWriter instances in this list, but shouldn't we also check for the list itself (rowOutWriters) being null here?

Reply:
You're right, I added a null check for the rowOutWriters as well.